v2/client.go

```diff
@@ -0,0 +1,56 @@
+package v2 // import "go.mongodb.org/atlas/mongodbatlas/v2"
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"runtime"
+
+	"github.com/mongodb-forks/digest"
+	apiv2latest "go.mongodb.org/atlas/mongodbatlas/v2/api"
+)
+
+const (
+	// CloudURL is default base URL for the services.
+	CloudURL = "https://cloud.mongodb.com/"
+	// Version the version of the current API client. Should be set to the next version planned to be released.
+	// TODO use common version across entire SDK
+	Version = "0.21.0"
```

Comment:
let's go there
```suggestion
	Version = "0.99.0"
```

Reply:
WHY?
I got information that for our sanity we need to flag or prefix version to differentiate between old and new SDK. 
That means that the client name should change and the version should be different. 

Removing TODO